lib/read.js

```diff
@@ -239,7 +238,7 @@ class ReadSql extends AdapterRead {
         var timeField = this.timeField || 'time';
         var buckets = this.getBatchBuckets(from, to);
 
-        return Promise.map(buckets, (bucket, i, len) => {
+        return Promise.mapSeries(buckets, (bucket, i, len) => {
```

Comment:
I can bring concurrency back. But here is what I would need to do: worth it? https://github.com/juttle/juttle-sql-adapter-common/pull/47